srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/AdminHomeController.java

```diff
@@ -37,6 +47,19 @@ public AdminHomeController(final AdminService adminService)
     @RequestMapping(method = RequestMethod.GET)
     public String defaultPage(final Model model)
     {
+        final ImmutableCollection<RiskModel>  riskModels = fAdminService.getAllRiskModels();
+
+        // Use a set since there are multiple entries in the risk models list for each model.
+        //
+        Set<EditRiskModel> editRMs = new TreeSet<EditRiskModel>(  );
```

Comment:
My previous comment still stands: I see no reason to translate from RiskModels to EditRiskModels here. We can just use RiskModels as the models for the view.